app/models/district.rb

```diff
@@ -47,6 +47,13 @@ def unconvinced_legislators
     senators.eligible.unconvinced + [unconvinced_rep].compact
   end
 
+  def legislators
+    join_clause = 'INNER JOIN districts '\
+                  'ON (districts.id = legislators.district_id '\
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.